src/allotropy/parsers/utils/pandas.py

```diff
@@ -80,6 +81,14 @@ def assert_value_from_df(df: pd.DataFrame, key: str) -> Any:
         raise AllotropeConversionError(msg) from e
 
 
+def _normalize_columns(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
I feel like I just reviewed these changes, right? (will probably be fixed once the other PR merges in first, I assume just some commit got mixed in)

Reply:
Oh dang yep, must not have switched back to main before opening this up. That merged, will update.